code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -254,3 +254,32 @@
 		rad = 0
 	)
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS|LEGS
+
+/obj/item/clothing/suit/storage/bladerunner
+	name = "leather coat"
+	desc = "Old leather coat. Probably saw things you wouldn't believe"
```

Comment:
"An old leather coat. Probably seen things you wouldn't believe." or "This coat has probably seen things you wouldn't believe."